exonum-light-client/src/main/java/com/exonum/client/response/HealthCheckInfo.java

```diff
@@ -15,14 +15,12 @@
  *
  */
 
-package com.exonum.client;
+package com.exonum.client.response;
 
 import lombok.Value;
 
-/**
- * Json object wrapper for memory pool response.
- */
 @Value
-class MemoryPoolResponse {
-  int size;
+public class HealthCheckInfo {
+  ConsensusStatus consensusStatus;
+  int connectionsNumber;
```

Comment:
Shall we enforce any restrictions on the values?

Reply:
do you mean negative values? Also, we have some restriction for a number of network nodes. But I'm not sure that LC should know about theoretical maximum of that count